src/format.iced

```diff
@@ -0,0 +1,122 @@
+stream = require('keybase-chunk-stream')
+util = require('./util')
+
+space = Buffer.from(' ')
+newline = Buffer.from('\n')
+punctuation = Buffer.from('.')
+
+words_per_line = 200
+chars_per_word = 15
+
+noop = () ->
+
+exports.FormatStream = class FormatStream extends stream.ChunkStream
+
+  __format : (chunk) ->
+    return Buffer.concat([space, chunk])
+
+  _format : (chunk) ->
+    if chunk.length < chars_per_word then return chunk
+    results = []
+    for i in [0...chunk.length] by chars_per_word
+      word = chunk[i...i+chars_per_word]
+      if i+chars_per_word >= chunk.length
+        results.push(word)
+      else
+        if @_word_count % 200 is 0 and @_word_count isnt 0 then word = Buffer.concat([word, newline])
+        else word = Buffer.concat([word, space])
```

Comment:
Since we have conditional lines followed by unconditional lines, lets use the regular

```
if a
  b
else
  c
d
e
```

indentation style to distinguish them.